java/src/main/java/org/rocksdb/ComparatorType.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+package org.rocksdb;
+
+enum ComparatorType {
+  JAVA_COMPARATOR((byte)0x0),
```

Comment:
Nit: Could you add a short 1-line comment for each of these? I was jumping back to the corresponding *Comparator.java class javadocs initially, a brief description would help the reader.